tests/unit/test_types.py

```diff
@@ -241,6 +241,39 @@ def test_struct_field_name():
     assert sf.column_identifier.name == "integer type"
 
 
+def test_struct_get_item():
+    sfa = StructField("a", IntegerType())
+    sfb = StructField("b", StringType())
+    sfc = StructField("c", LongType())
+
+    st = StructType([sfa, sfb, sfc])
```

Comment:
nit: maybe name it `struct_type`? I don't have a strong opinion on this one.

Reply:
Done